en_US/data-integration/data-bridge-kafka.md

```diff
@@ -54,54 +59,151 @@ bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c config/kraft/server.properti
+     
+     **Note**: Each Kafka-to-MQTT topic mapping must contain a unique Kafka topic name.  That is, the Kafka topic must not be present in more than one mapping.
+     
+   
+   :::
+   
+   ::::
+   
+5. Before clicking **Create**, you can click **Test Connection** to test that the bridge can connect to the Kafka server.
+
+6. Click **Create**, you'll be offered the option of creating an associated rule. 
+
+   - For the Kafka producer data bridge, click **Create Rule** or go to [Create Rule for Kafka Producer Data Bridge](#create-rule-for-kafka-producer-data-bridge) to create an associated rule. 
+   - For the Kafka consumer data bridge, It's not strictly necessary to create a rule.
 
 ::: tip
-Currently EMQX only supports creating data bridges to Kafka in **Producer** mode. 
+
+   By creating a rule, it allows Kafka messages matching the rule to be further transformed and filtered if needed, and then forwarded to other rule actions, like different bridges. Refer to the [Rules](./rules.md) for more information on creating rules. The MQTT topics defined in **Topic Mapping** will start having messages published to them without further configuration.
+
 :::
 
-### Test
+Now the Kafka data bridge should appear in the data bridge list (**Data Integration** -> **Data Bridge**) with **Resource Status** as **Connected**.
 
-Use MQTTX to send messages to topic  `t/1`:
+### Create Rule for Kafka Producer Data Bridge
 
-```bash
-mqttx pub -i emqx_c -t t/1 -m '{ "msg": "Hello Kafka" }'
+1. Go to EMQX Dashboard, click **Data Integration** -> **Rules**.
+
+2. Click **Create** on the top right corner of the page.
+
+3. Input, for example, `my_rule` as the rule ID.
+
+4.  Input the following statement in the **SQL Editor** if you want to save the MQTT messages under topic `t/#`  to Kafka. 
+
+   Note: If you want to specify your own SQL syntax, make sure that you have included all fields required by the data bridge in the `SELECT` part.
+
+```sql
+SELECT
+  *
+FROM
+  "t/#"
 ```
 
-Check the running status of the two data bridges, there should be one new incoming and one new outgoing message. 
+5. Click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list and then select the data bridge you just created under **Data bridge**. Then click the **Add** button.
 
-Check whether messages are written into the topic `testtopic-in`  with the following Kafka command:
+6. Click **Create** at the page bottom to finish the creation.
+
+Now we have successfully created the data bridge to Kafka producer data bridge. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `t/#`  are sent and saved to Kafka after parsing by rule  `my_rule`.
```

Comment:
```suggestion
Now you have successfully created the data bridge to Kafka producer data bridge. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `t/#`  are sent and saved to Kafka after parsing by rule  `my_rule`.
```